src/robusta/integrations/slack/sender.py

```diff
@@ -189,6 +189,15 @@ def __send_blocks_to_slack(
         )
         other_blocks = [b for b in report_blocks if not isinstance(b, FileBlock)]
 
+        # wide tables aren't displayed properly on slack. looks better in a text file
+        table_blocks = [b for b in other_blocks if isinstance(b, TableBlock)]
+        for table_block in table_blocks:
+            if len(table_block.headers) > 4:
```

Comment:
I would make the 4 something like MAX_HEADERS_AMNT_FOR_SLACK_TABLE=4 at the top of the file to make it easier to find and change in the future

Reply:
fixed